conanfile.py

```diff
@@ -254,6 +254,8 @@ def build(self):
             args.insert(0, "-shared")
         if self.settings.build_type == "Debug":
             args.append("-debug")
+        elif self.settings.build_type == "Release":
+            args.append("-release")
```

Comment:
Please remove `args.append("-release")` line 266 and instead throw an error, because it is in this case an unknown build_type